regression/cbmc/symex_should_filter_value_sets/test.desc

```diff
@@ -0,0 +1,25 @@
+CORE
+main.c
+--show-vcc
+^EXIT=0$
+^SIGNAL=0$
+main::1::c1!0@1#. = 1
+main::1::c2!0@1#. = 1
+main::1::c3!0@1#. = 1
+main::1::c4!0@1#. = 1
+main::1::c5!0@1#. = 1
+main::1::c6!0@1#. = 1
+main::1::c7!0@1#. = 1
+main::1::c8!0@1#. = 1
+main::1::c9!0@1#. = 1
+main::1::c10!0@1#. = 1
+main::1::c11!0@1#. = 1
+main::1::c12!0@1#. = 1
+main::1::c13!0@1#. = 1
+main::1::c14!0@1#. = 1
+--
+^warning: ignoring
+main::1::c15!0@1#. = 1
+main::1::c16!0@1#. = 1
+--
+
```

Comment:
Instead the the blank line you might want to explain why all the above patterns are the right things to test for? Even more so as this test is failing...

Reply:
There are some comments in `main.c`. You may still find them too sparse.

Reply:
It might be sufficient to say "See comments in main.c for explanations on why each of the assignments is (not) expected to appear in the output."